operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md

```diff
@@ -26,6 +26,13 @@ echo $XNAME
 
 Only follow the steps in the section for the node type that is being rebuilt.
 
+> **`NOTE:`** (`ncn#`) Restart the `goss-servers` service on the rebuilt node after it has been rebuilt.
+> This is necessary because of a timing issue that has been fixed in CSM 1.6.1.
+>
+>    ```bash
+>    ssh "${NODE}" 'systemctl restart goss-servers'
+>    ```
```

Comment:
I don't think this needs to be indented

````suggestion
> ```bash
> ssh "${NODE}" 'systemctl restart goss-servers'
> ```
````

Reply:
fixed